modules/administration-guide/partials/con_architecture-overview-with-che-server.adoc

```diff
@@ -0,0 +1,25 @@
+[id="{prod-id-short}-architecture-overview-with-che-server_{context}"]
+= {prod-short} architecture with {prod-id-short}-server
+
+{prod-short} server is the default workspace engine.
+
+.High-level {prod-short} architecture with the {prod-short} server engine
+image::administration-guide:architecture/{project-context}-architecture-with-che-server-engine.png[]
```

Comment:
In the Preview, this graphic renders with arrows over the top of some of the text. Maybe it's just my browser, but it doesn't look very clean to my eyes.